docs/commands/clip.md

```diff
@@ -0,0 +1,15 @@
+# clip
+
+Copy the contents of the pipeline to the copy/paste buffer
+
+
+# Syntax
+```cmd
+<command> clip
```

Comment:
The command is called `clip`. Why is there `<command> clip`? You only do `> clip` and that's it.